<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_edge_edit</class>
 <widget class="QWidget" name="Form_edge_edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>333</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>274</width>
    <height>333</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>274</width>
    <height>333</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>edit edge</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
background-color: rgb(233, 226, 255);
font-size: 14px;
font-family: &quot;System&quot;;
color: rgb(111, 31, 158);
}

QLineEdit
{
background-color: rgb(255, 255, 255);
border-radius: 8px;
}

QPushButton
{
background-color: rgb(233, 226, 255);
border: 2px solid rgb(111, 31, 158);
border-radius: 8px;
color: rgb(111, 31, 158);
}

QPushButton:hover
{
	background-color: rgb(176, 170, 255)
}

QPushButton:pressed
{
	background-color: rgb(152, 147, 220);
}


</string>
  </property>
  <widget class="QLabel" name="label_b">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>274</width>
     <height>333</height>
    </size>
   </property>
   <property name="text">
    <string>B:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>274</width>
     <height>333</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 14px;
font-family: &quot;System&quot;;</string>
   </property>
   <property name="text">
    <string>Edit edge</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ba">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>274</width>
     <height>333</height>
    </size>
   </property>
   <property name="text">
    <string>B --&gt; A :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ba">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>274</width>
     <height>333</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ab">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>274</width>
     <height>333</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_edit_edge">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>274</width>
     <height>333</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>EDIT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ab">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>274</width>
     <height>333</height>
    </size>
   </property>
   <property name="text">
    <string>A --&gt; B :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_a">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>274</width>
     <height>333</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_b">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>274</width>
     <height>333</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_a">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>274</width>
     <height>333</height>
    </size>
   </property>
   <property name="text">
    <string>A:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
